packages/api/.env

```diff
@@ -2,7 +2,7 @@ LP_API_BROADCASTERS=[{"address":"http://127.0.0.1:3085","cliAddress":"http://127
 LP_API_FALLBACK_PROXY=http://localhost:3008
 LP_API_JWT_SECRET=secretsecret
 LP_API_JWT_AUDIENCE=livepeer
-LP_API_CORS_JWT_ALLOWLIST=http://localhost:3000
+LP_API_CORS_JWT_ALLOWLIST=["http://localhost:3000"]
```

Comment:
nit: I don't know if the name `CORS_JWT_ALLOWLIST` is technically correct. CORS is a browser specific limitation/feature, having nothing to do with jwt

Reply:
So the JWT part is not from the CORS definition, but from our own implementation of our CORS rules. This allow-list only applies for requests authenticated with a JWT.

If you prefer, I can rename this to `LP_API_JWT_CORS_WHITELIST` instead. Do you think it makes more sense/

Reply:
oh nice! that clears up my doubts. I was just confused with the jwt being present in there.